walk-in-interview/src/main/webapp/script.js

```diff
@@ -256,6 +261,22 @@ function buildJobElement(job) {
     detailsForm.submit();
   });
 
+  const marker = addMarker(map, job);
+  marker.addListener('click', function() {
+    new google.maps.InfoWindow({
+      content: jobListing.innerHTML,
```

Comment:
* does the jobListing mean one job post listed on the home page? maybe rename to jobPost?
* can u take a screenshot of how it looks like? i am not sure if it is user-friendly to contain all info appearing on job post on homepage in a small info window, especially this web is viewed on mobile phone screen

Reply:
agreed. maybe you need a job preview content.

Reply:
I renamed it to jobPostPreview. Here's what is looks like:
![image](https://user-images.githubusercontent.com/55294931/88757671-2d64be80-d199-11ea-93c2-0b25aa8d304f.png)